src/traits.rs

```diff
@@ -62,7 +62,7 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
         for (i, method) in ::rustc::traits::get_vtable_methods(self.tcx, trait_ref).enumerate() {
             if let Some((def_id, substs)) = method {
-                let instance = ::eval_context::resolve(self.tcx, def_id, substs);
+                let instance = resolve(self.tcx, def_id, substs);
```

Comment:
I wouldn't do this, since these are copied from librustc_trans. I'd rather replace global paths with importing the *module* and so you get to remove `::`.